src/main/java/plus/misterplus/ivrench/InvertedEnchantments.java

```diff
@@ -14,6 +18,11 @@
     public static final String VERSION = "1.0.3";
 
     private static Logger logger = LogManager.getLogger(MOD_NAME);
+    public final static CreativeTab TAB = new CreativeTab();
```

Comment:
change to
`public final static CreativeTab MOD_TAB = new CreativeTab();`

Reply:
got it